spec/cassettes/OpenTok_Archives/when_many_archives_are_created/should_return_session_archives.yml

```diff
@@ -0,0 +1,71 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://api.opentok.com/v2/project/123456/archive?sessionId=SESSIONID
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      X-Tb-Partner-Auth:
```

Comment:
Can you replace `X-Tb-Partner-Auth` with `X-Opentok-Auth` like the other cassettes. See: https://github.com/opentok/OpenTok-Ruby-SDK/blob/master/spec/cassettes/OpenTok_Archives/should_create_archives.yml#L12